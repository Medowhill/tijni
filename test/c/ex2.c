#include <stdlib.h>

int *f(int x) {
  int *p = malloc(sizeof(int));
  *p = x;
  return p;
}
