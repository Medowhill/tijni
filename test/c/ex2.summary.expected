{
  "f": {
    "params": [ { "name": "%x(f)", "used": false, "freed": false } ],
    "ret": [ "Allocsite(0)" ],
    "env": {
      "%x(f)": [ "'%x(f)" ],
      "%call(f)": [ "Allocsite(0)" ],
      "%0(f)": [ "Allocsite(0)" ],
      "Allocsite(0)": [ "'%x(f)" ]
    }
  }
}